{"name": "Unsuccessful login", "status": "passed", "steps": [{"name": "Error message is visible", "status": "passed", "start": 1737703770357, "stop": 1737703770357}], "parameters": [{"name": "username", "value": "'standard_use'"}, {"name": "password", "value": "'secret_sauce'"}, {"name": "driver", "value": "'Edge'"}], "start": 1737703770035, "stop": 1737703770398, "uuid": "c4679f58-c906-4c95-b371-a08853528d5c", "historyId": "86bc5338a5c26c00a62dcfde57e10d39", "testCaseId": "b876685be5aab7257716ff999cc789c0", "fullName": "Tests.test_Login.TestLogin#test_1", "labels": [{"name": "feature", "value": "Login"}, {"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "fv-az2032-218"}, {"name": "thread", "value": "1943-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login"}]}